server/src/test/proto/spine/test/procman/commands.proto

```diff
@@ -52,3 +53,11 @@ message PmDoNothing {
 message PmThrowEntityAlreadyArchived {
     ProjectId project_id = 1;
 }
+
+message ArchivePm {
```

Comment:
`PmArchiveProcess`. `PmDeleteProcess` below.

Let's not violate our conventions.